<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>947</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widgetLeftSide" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btOpenDir">
           <property name="text">
            <string>Open directory...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="chkRecursive">
           <property name="text">
            <string>recursive</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btReload">
           <property name="text">
            <string>Reload</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="listFiles">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widgetRightSide" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupSettings">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QRadioButton" name="rbPrefixNew">
            <property name="text">
             <string>add new prefix</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QRadioButton" name="rbPrefixExisting">
            <property name="text">
             <string>use existing prefix</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="lbSeparator">
            <property name="text">
             <string>Separating character:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="edSeparator">
            <property name="text">
             <string notr="true">_</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="lbFileCount">
            <property name="text">
             <string>File count:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="lbFileCountValue">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="lbNoDigits">
            <property name="text">
             <string>Number of digits:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="spinNoDigits"/>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="chkNewInsideFolders">
            <property name="text">
             <string>new numbering inside folders</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widgetExample" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="lbExample">
            <property name="text">
             <string>Example:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbExampleValue">
            <property name="text">
             <string>asdf</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupSubDirSettings">
         <property name="title">
          <string>Subdirectory settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="3" column="1">
           <widget class="QLineEdit" name="edDirectoryPrefix"/>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="chkCreateSubDirs">
            <property name="text">
             <string>create subdirectories</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lbFilesPerDir">
            <property name="text">
             <string>Files per directory:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" rowspan="2">
           <widget class="QSpinBox" name="spinFilesPerDir"/>
          </item>
          <item row="2" column="0" rowspan="2">
           <widget class="QLabel" name="lbDirectoryPrefix">
            <property name="text">
             <string>Directory prefix:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="chkDeleteOldDirs">
            <property name="text">
             <string>delete old directories</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>chkCreateSubDirs</zorder>
         <zorder>lbFilesPerDir</zorder>
         <zorder>spinFilesPerDir</zorder>
         <zorder>lbDirectoryPrefix</zorder>
         <zorder>edDirectoryPrefix</zorder>
         <zorder>verticalSpacer</zorder>
         <zorder>chkDeleteOldDirs</zorder>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>45</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btRename">
         <property name="text">
          <string>Rename</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>947</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
